lib/wanda_web/views/v1/catalog_view.ex

```diff
@@ -13,10 +13,19 @@ defmodule WandaWeb.V1.CatalogView do
 
   def adapt_v1(%{expectations: expectations} = check) do
     adapted_expectations =
-      Enum.map(expectations, fn
+      expectations
+      |> Enum.map(fn
         %{type: :expect_enum} = expectation -> Map.put(expectation, :type, :unknown)
         expectation -> expectation
       end)
+      |> Enum.map(fn %{
```

Comment:
what about:
```
|> Enum.map(&(Map.drop(&1, [:warning_message]))
```
At the end, we want to remove that field, right? It makes it more obvious